### PR TITLE
Component calc with class

### PR DESCRIPTION
**Pull request by GintsM**
Created following 
 - directory `components`
 - class based component `Calculator`
 - no operations are available yet
 - styling with CSS

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <p>Ready for Work</p>
-    </div>
-  );
-}
+const App = () => (
```
Comment (Mihndim2020) [inline src/App.js:3]: - [ ] As it is a requirement for this project, your `App` component should be `class-based`. Please kindly fix that.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+## Built With
+
+- REACT
+- CSS
+- HTML
+
 # Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (Mihndim2020) [inline README.md:13]: - [ ] Please it will be nice to add your personal and license information on the `readme` file.

File: src/index.css [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+  border: none;
+  margin-top: 20px;
+}
+
+.input-field {
+  background-color: #848692;
+  height: 50px;
+  width: 100%;
+}
+
```
Comment (Mihndim2020) [inline src/index.css:40]: - [ ] The display area of your calculator is overlapping, please see image below and kindly fix it

calcDisplay
.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @GintsM, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below under the review._
- [ ] Please delete all unnecessary files(texts, images, and styles) created by `CRA`, check your public directory as this will make your repository cleaner. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (DeliceLydia) [reply README.md:21]: Solved this

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @GintsM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (DeliceLydia) [reply src/App.js]: Solved it

**Pull request merged**